workflow/validate/validate_test.go

```diff
@@ -1495,13 +1495,49 @@ spec:
         path: /mnt
 `
 
+var nonPathOutputParameter = `
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: non-path-out-param-
+spec:
+  entrypoint: non-path-out-param
+  templates:
+  - name: non-path-out-param
+    steps:
+    - - name: non-path-resource-out-param
+        template: non-path-resource-out-param
+    outputs:
+      parameters:
+      - name: param
+        valueFrom:
+          parameter: p
```

Comment:
Can we fill this in with a real parameter? Having complete workflows in our test cases makes them readily available when needed

Reply:
Fixed it. Can you review it again?